meinberlin/assets/scss/components/_rating.scss

```diff
@@ -39,12 +41,20 @@
         color: contrast-color($color);
     }
 
+    .fa {
+        margin-right: 0.3em;
+    }
+
     &.is-read-only,
     &:disabled {
         color: $color;
         background: none;
         border: 0;
     }
+
+    .comment & {
+        padding: 0.3em 0.6em;
+    }
```

Comment:
Ratings are render via react and ratings for comments are supposed to be smaller by design. So even if we'd use a modifier we'd have to alter the core.